VENCalculatorInputView/VENMoneyCalculator.m

```diff
@@ -63,6 +63,7 @@ - (NSNumberFormatter *)numberFormatter {
         _numberFormatter = [NSNumberFormatter new];
         [_numberFormatter setLocale:self.locale];
         [_numberFormatter setNumberStyle:NSNumberFormatterCurrencyStyle];
+        [_numberFormatter setMaximumFractionDigits:2];
```

Comment:
Given a currency formatter + currency Code, the maximumFranctionDigits as well as minimum is applied.

Here, it appears like we're massaging this one to play friendly with our expectations and I wondering if we're misunderstanding something here?

Could you add some more details about the missing decimal use case you're seeing?